src/status_im/ui/screens/routing/routes.cljs

```diff
@@ -0,0 +1,151 @@
+(ns status-im.ui.screens.routing.routes
+  (:require
+   [status-im.ui.screens.main-tabs.views :as main-tabs]
+
+   [status-im.ui.screens.accounts.login.views :refer [login]]
```

Comment:
The old code was using refer as well but shouldn't we use :as for new code

Reply:
technically that's a copied/moved old code. But ok